src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets/ignitus-Icons/remoteJobIcon/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { HTMLAttributes } from 'react';
```

Comment:
Remote, location icons look odd, where did you take from?
You should add from Material design icons rounded ones.